packages/fast-components-react-base/src/viewport-positioner/viewport-positioner.props.ts

```diff
@@ -58,6 +58,11 @@ export interface ViewportPositionerHandledProps extends ViewportPositionerManage
      */
     horizontalAlwaysInView?: boolean;
 
+    /**
+     *  Locks horizontal axis to default position
+     */
+    horizontalLocktoDefault?: boolean;
```

Comment:
"T" for "to" should be capitalized `horizontalLockToDefault`

Reply:
yep